app/services/flow/steps/providers_home_step.rb

```diff
@@ -0,0 +1,7 @@
+module Flow
+  module Steps
+    ProvidersHomeStep = Step.new(
+      path: ->(_application) { urls.providers_legal_aid_applications_path },
```

Comment:
Can we use the same pattern for paths that don't need an application?

either `->(_application)` or ` ->(_)` that was used on applicants_step